modules/runtime/nuxeo-runtime-metrics/src/main/java/org/nuxeo/runtime/metrics/reporter/StackdriverReporter.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     bdelbosc
+ */
+package org.nuxeo.runtime.metrics.reporter;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Set;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.nuxeo.common.utils.DurationUtils;
+import org.nuxeo.runtime.metrics.AbstractMetricsReporter;
+
+import io.dropwizard.metrics5.MetricAttribute;
+import io.dropwizard.metrics5.MetricFilter;
+import io.dropwizard.metrics5.MetricRegistry;
+import io.opencensus.common.Duration;
+import io.opencensus.contrib.dropwizard5.DropWizardMetrics;
+import io.opencensus.exporter.stats.stackdriver.StackdriverStatsConfiguration;
+import io.opencensus.exporter.stats.stackdriver.StackdriverStatsExporter;
+import io.opencensus.metrics.Metrics;
+
+/**
+ * Reports metrics to Google Stackdriver.
+ *
+ * @since 11.4
+ */
+public class StackdriverReporter extends AbstractMetricsReporter {
+
+    private static final Logger log = LogManager.getLogger(StackdriverReporter.class);
+
+    protected static final String PREFIX_OPTION = "prefix";
+
+    protected static final String DEFAULT_PREFIX = "custom.googleapis.com/nuxeo/";
+
+    protected boolean activated;
+
+    @Override
+    public void start(MetricRegistry registry, MetricFilter filter, Set<MetricAttribute> deniedExpansions) {
+        log.warn("Creating Stackdriver metrics reporter");
```

Comment:
`warn` -> `debug`

Reply:
All reporters state their activity at warning level, this is something that we don't want to miss when doing support.